pkg/build/build.go

```diff
@@ -91,9 +90,7 @@ func writeLicense(manifest model.Manifest) error {
 	cmd.Stdout = o
 	cmd.Stderr = os.Stderr
 	if err := cmd.Run(); err != nil {
-		// TODO fail hard
-		log.Errorf("failed to get license: %v", err)
-		return nil
+		return fmt.Errorf("failed to get license: %v", err)
```

Comment:
Perhaps "unable to generate license repo for repo %s: %v" would be more descriptive and helpful?